### PR TITLE
Display/search

### PR DESCRIPTION
**Pull request by LaiYW11**
Added the display section. help check

### CODE REVIEW

File: menus/display_menu.py [hunk 1]
```diff
@@ -69,48 +69,50 @@ def selection(self):
...
-                if search_term =='exit'.lower():
+                # if user enters exit, program will return to main menu
+                if search_term == 'exit':
                     self.root.display().selection()
 
-                #for every row in data, if there are any terms matching the items in rows, add the entire row to a new list
+                # for every row in data, check if search term matches ISBN, AUTHOR, or TITLE
                 for row in data:
-                    if any(search_term in str(item).lower() for item in row):
+                    isbn, author, title, status = row[:4]  #to allow search function for elements ISBN, AUTHOR, TITLE ONLY
```
Comment (LaiYW11) [inline menus/display_menu.py]: I made a mistake here. It's supposed to be isbn, author, title = row[0:3]. I will commit my final code once I fix one of the errors I am having and make a pull request.

Comment (Spimy) [reply menus/display_menu.py]: The pull request is already open just make the necessary changes and I'll give it a review.

File: menus/display_menu.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    # check if the search term is in ISBN, AUTHOR, or TITLE
+                    if search_term in isbn.lower() or search_term in author.lower() or search_term in title.lower():
+                        searched_books.append(row)
+
+                # checks whether searched_books list is not empty
+                if len(searched_books) > 0:
+                    print("\nSearch Results:")
+                    
+                    # print headers with '-' to make it look cleaner
+                    print(header_line)
```
Comment (Spimy) [inline menus/display_menu.py]: Notice how the code onward is basically the same as displaying all books for tabulation? Try creating a method (function) specifically for displaying to reduce duplicated code.

File: menus/display_menu.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            user_choice = input("Invalid Input, Search Books[Y] or Exit to Main Menu[N]: ").lower()
+
+        if user_choice == 'y':
+            while True:
+                search_term = input("Search book by ISBN, AUTHOR, or TITLE (Type 'EXIT' to return to main menu): ").lower()
+                searched_books = []
+
+                # if user enters exit, program will return to main menu
+                if search_term == 'exit':
+                    self.root.display().selection()
```
Comment (Spimy) [inline menus/display_menu.py]: Because you are in a while true loop, make sure you break after you display the main menu.

**Review by Spimy — CHANGES_REQUESTED** [review]

**Comment by Spimy** [comment]
@JBBru-helloworld Can you review the code so that it can be merged to the main branch?

**Review by Spimy — APPROVED** [review]

**Review by Spimy — APPROVED** [review]

**Review by Spimy — APPROVED** [review]

**Review by JBBru-helloworld — APPROVED** [review]

**Pull request merged**